sdk/storage/azure-storage-files-shares/CMakeLists.txt

```diff
@@ -61,7 +61,6 @@ target_include_directories(
     PUBLIC
       $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/inc>
       $<INSTALL_INTERFACE:include>
-      ${azure-storage-common-cpp_INCLUDE_DIRS}
```

Comment:
Don't remove. Does it break any builds?

Reply:
duplicate, discuss in another conversation